Earthfile

```diff
@@ -155,9 +155,17 @@ go-build:
   CACHE --id go-build --sharing shared /root/.cache/go-build
   COPY --dir apis/ cmd/ internal/ pkg/ .
   RUN go build -o crossplane ./cmd/crossplane
+  RUN sha256sum crossplane | head -c 64 > crossplane.sha256
   RUN go build -o crank ./cmd/crank
+  RUN sha256sum crank | head -c 64 > crank.sha256
+  RUN tar -czvf crank.tar.gz crank crank.sha256
+  RUN sha256sum crank.tar.gz | head -c 64 > crank.tar.gz.sha256
   SAVE ARTIFACT crossplane AS LOCAL _output/bin/${GOOS}_${GOARCH}/crossplane
+  SAVE ARTIFACT crossplane.sha256 AS LOCAL _output/bin/${GOOS}_${GOARCH}/crossplane.sha256
   SAVE ARTIFACT crank AS LOCAL _output/bin/${GOOS}_${GOARCH}/crank
+  SAVE ARTIFACT crank.sha256 AS LOCAL _output/bin/${GOOS}_${GOARCH}/crank.sha256
+  SAVE ARTIFACT crank.tar.gz AS LOCAL _output/bundle/${GOOS}_${GOARCH}/crank.tar.gz
```

Comment:
is there particular "standard" guidance that helped you land on `bundle` as the name of this dir? curious to see if there's an already defined practice we're following :)

Reply:
I don't think there is any standard naming applied here, and I don't really have a preference on how to name the dir. I chose bundle because this is what `up` uses 😉 https://cli.upbound.io/stable/v0.30.0/bundle